crypto-primitives/src/commitment/pedersen/constraints.rs

```diff
@@ -98,6 +98,16 @@ where
     W: PedersenWindow,
     ConstraintF: PrimeField,
 {
+    fn alloc_constant<T, CS: ConstraintSystem<ConstraintF>>(
+        cs: CS,
+        val: T,
+    ) -> Result<Self, SynthesisError>
+    where
+        T: Borrow<PedersenParameters<G>>,
+    {
+        Self::alloc(cs, || Ok(val))
+    }
```

Comment:
This is okay because there's nothing to allocate, right?

Reply:
Exactly, that was the thought.
Maybe we should reverse the order though - move the code to alloc_constant and have alloc call that, for clarity :)

Reply:
Yeah, that's probably better =)